custom/icds_reports/sqldata.py

```diff
@@ -275,13 +291,9 @@ def domain(self):
 
     @property
     def filters(self):
-        filters = [
-            EQ('aggregation_level', 'aggregation_level')
-        ]
+        filters = super(AggChildHealthMonthlyDataSource, self).filters
```

Comment:
This class's loc_key is still `%_site_code`

Reply:
Updated